test/snapshots/blueprint.test.ts.md

```diff
@@ -270,23 +275,23 @@ Generated by [AVA](https://avajs.dev).
                   code: {
                     javascript: {
                       request: 'await seam.foos.list()',
-                      response: '[{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo"}]',
+                      response: '[{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo","metadata":null}]',
                       title: 'JavaScript',
                     },
                     php: {
                       request: '$seam->foos->list()',
-                      response: '[{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo"}]',
+                      response: '[{"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo","metadata":null}]',
                       title: 'PHP',
                     },
                     python: {
                       request: 'seam.foos.list()',
-                      response: '[Foo(foo_id="8d7e0b3a-b889-49a7-9164-4b71a0506a33", name="Best foo")]',
+                      response: '[Foo(foo_id="8d7e0b3a-b889-49a7-9164-4b71a0506a33", name="Best foo", metadata=None)]',
                       title: 'Python',
                     },
                     ruby: {
                       request: 'seam.foos.list()',
                       response: `<Seam::Foos:0x00000␊
-                        0={"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo"}>`,
+                        0={"foo_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33","name":"Best foo","metadata":null}>`,
```

Comment:
I thought Ruby uses `nil`?

Reply:
Yes, it does. The array responses are broken, that's why it has `0={...}`. 
https://github.com/seamapi/blueprint/pull/54 fixes this

Reply:
I merged the mentioned PR and now the ruby list samples are correct.